DEVELOPMENT.md

```diff
@@ -127,9 +127,14 @@ go test -v ./... -tags=integration
 Use [delve](https://golangforall.com/en/post/go-docker-delve-remote-debug.html)
 to start a debugger to which attach from your IDE.
 
-1. Build the dependencies of the chart
+1. Install knative-events and cert-manager operators
    ```bash
-   helm dependency build charts/kubearchive
+   export CERT_MANAGER_VERSION=v1.9.1
+   export KNATIVE_EVENTING_VERSION=v1.14.3
+  
+   kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${CERT_MANAGER_VERSION}/cert-manager.yaml 
+   kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-crds.yaml
+   kubectl apply -f https://github.com/knative/eventing/releases/download/knative-${KNATIVE_EVENTING_VERSION}/eventing-core.yaml
```

Comment:
Why is this inside the "remote IDE debugging" section? This should be after the cluster creation.

Reply:
yes you are right, should be fixed now